### PR TITLE
Added Some commands in git cheatsheet

### PR DESCRIPTION
**Pull request by nikhilbuts**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [ ] Search for duplicates.
- [ ] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/git-cheatsheet.md [hunk 1]
```diff
@@ -92,6 +92,8 @@ Table of Contents
 | `git push origin [branch name]` | Push a branch to your remote repository |
 | `git pull origin [branch name]` | Pull a branch from your remote repository |
 | `git remote origin [branch name]` | Connect repository to local server |
+| `git push -f` | force a push request
```
Comment (crescentpartha) [inline CheatSheets/git-cheatsheet.md:95]:
```suggestion
| `git push <remote> --force` | Forces the push even if it results in a non-fast-forward merge. Be sure that nobody has pulled the commits before using the `--force` option. |
```
| `git push <remote> --force` | Forces the push even if it results in a non-fast-forward merge. Be sure that nobody has pulled the commits before using the `--force` option. |

Need to `mention the <remote> branch`.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Request for changes.

File: CheatSheets/git-cheatsheet.md (hunk 1 above)
Comment (nikhilbuts) [reply CheatSheets/git-cheatsheet.md]: Ok. @crescentpartha 

Thanks for Suggestion. I am changing it.

**Review by crescentpartha — APPROVED** [review]
Approved.

**Pull request merged**